src/webview/js/js.js

```diff
@@ -637,16 +628,23 @@ const handleLongPress = (target: Element) => {
   // a library we can use which strives to handle all that complexity, or
   // get long-press events from the platform.
   if (!lastTouchEventTimestamp || Date.now() - lastTouchEventTimestamp < 500) {
-    return;
-  }
-
-  lastTouchEventTimestamp = 0;
+    if (target.matches('a')) {
+      sendMessage({
+        type: 'url',
+        href: target.getAttribute('href'),
+        messageId: getMessageIdFromNode(target),
+      });
+    }
+  } else {
+    lastTouchEventTimestamp = 0;
 
-  sendMessage({
-    type: 'longPress',
-    target: target.matches('.header') ? 'header' : 'message',
-    messageId: getMessageIdFromNode(target),
-  });
+    sendMessage({
+      type: 'longPress',
+      target: target.matches('.header') ? 'header' : target.matches('a') ? 'link' : 'message',
+      messageId: getMessageIdFromNode(target),
+      href: target.matches('a') ? target.getAttribute('href') : '',
```

Comment:
Let's not use `''` as a symbol for "no string here" -- it should only be used when we really want to say there's a string which is empty.

Instead, `null` is good. (Or `undefined`, but that doesn't translate through JSON.)

Reply:
Done! :)